oapen-engine/Dockerfile

```diff
@@ -27,15 +28,26 @@ ENV PATH="$PATH:/usr/local/bin"
 
 
 RUN apt-get update && apt-get install libpq5 -y
+
 # Create and switch to a new user
 RUN useradd --create-home appuser
 WORKDIR /home/appuser
 USER appuser
 
+RUN [ "python", "-c", "import nltk; nltk.download('punkt', download_dir='/home/appuser/nltk_data'); nltk.download('stopwords', download_dir='/home/appuser/nltk_data')" ]
+
 ENV PYTHONPATH="/home/appuser/src"
 
 # Install application into container
 COPY . .
 
+USER root
+
+RUN chmod u+x scripts/clean.sh
```

Comment:
do we want to refresh on every deploy?

Reply:
I added some scripts for manual refresh/suggestion generation, but there's a flag in docker-compose.yml to set whether or not to refresh on deploy

Reply:
oh awesome